rest/handler.go

```diff
@@ -755,6 +769,12 @@ func (h *handler) checkPublicAuth(dbCtx *db.DatabaseContext) (err error) {
 			}
 		} else {
 			dbCtx.DbStats.Security().AuthSuccessCount.Add(1)
+
+			if h.isGuest() {
+				h.rqCtx = base.UserLogCtx(h.ctx(), base.GuestUsername, base.UserDomainSyncGateway)
+			} else if h.user != nil {
```

Comment:
What's the else case here? Is for a handler that doesn't need a user? Should this be logged?

Reply:
I think this is leftovers from when this code was in a different place. Reasonably confident I can remove the if